### ISSUE TITLE
Nexus Player, Oreo, SPMC and Terrible Performance 

### ISSUE THREAD

**Issue by mattspew** [koying/SPMC#1097]
Ever since upgrading to Oreo I get continued playback issue on 720 and up video; dropping frames and sometimes dropping the audio.

I've tried SPMC 16.6, 16.7, 16.7.1, 17.4a11 and Kodi 17.5; all versions work for after a reboot, but if you go away and come back it starts dropping frames like there's no tomorrow. I've also tried a factory reset and clean install of 17.4a11 and Kodi 17.5.

I've come to the conclusion Oreo is a hunk of crap and am on the verge of blowing it away and down-grading.

Anyone else using the same combo of hardware and software?

**Comment by nelsvieira** [comment]
Yes, I notice exactly the same thing. Cannot reproduce on Fire TV Stick. At this point I'm seriously considering using the Fire TV as my daily driver. I don't think Google has much love left for the Nexus Player.

**Comment by mattspew** [comment]
I've since updated to SPMC-spmc-krypton-1f9a682-armeabi-v7a.apk and the frame-dropping issue seems to have gone away, however the audio dropping and sometimes slow to return after pausing.

It's a real shame; the Nexus Player ticks all the boxes for ease, price, power, size and so on.

**Comment by koying** [comment]
Why not just revert the Nexus to nougat, where I understand you had no
issues?

**Comment by mattspew** [comment]
Fair point; however to be honest my paid streaming services work perfectly on the Nexus Player and I'm using SPMC/Kodi less and less as I torrent less and less.

A restart of the player is much quicker than downgrading and setting everything up again.

**Comment by koying** [comment]
Fair enough, but don't expect me to hook and support a basically dead
device on a semi-pre release android version that probably won't get any
support from Google anymore ;)
